src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -90,6 +90,7 @@ public class NetworkModificationService {
     private static final String DELIMITER = "/";
 
     private static final String NETWORK_MODIFICATION_TYPE_REPORT = "NetworkModification";
+    private static final String LINE_ID = "lineId";
```

Comment:
?

Reply:
I added this for the code smell : the "lineId" is used many times in createSubReporter calls for the different line creations